omod/src/main/webapp/resources/scripts/jquery-ui/js/openmrsSearch.js

```diff
@@ -851,14 +851,15 @@ function OpenmrsSearch(div, showIncludeVoided, searchHandler, selectionHandler,
                     attributeValue = rowData.attributes[a.name];
                     if(attributeValue == null)
                         attributeValue = '';
-
                     rRowData.push(attributeValue);
                 });
             }
-
+            for (k in rRowData)
+            {
+                rRowData[k] = $j("<div>").text(rRowData[k]).html();
```

Comment:
Isn't this change part of https://github.com/openmrs/openmrs-module-legacyui/pull/141?

Let's not mix PRs or else we'll have to resolve conflicts when they both get merged in.